src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div id="company-provisions">
+    <confirm-dialog ref="confirmCompanyProvisionsDialog" attach="#company-provisions" />
+    <v-layout v-if="!isEditing">
```

Comment:
v-layout and v-flex are deprecated.
Use v-container, v-row and v-col instead.

https://www.google.com/search?q=vuetify+v-layout+deprecated
https://vuetifyjs.com/en/getting-started/upgrade-guide/#grid
https://vuetifyjs.com/en/components/grids/#v-row

Reply:
(Flexbox helper classes are still OK I think.)

https://vuetifyjs.com/en/styles/flex/#flex